examples/example3/models.py

```diff
@@ -0,0 +1,34 @@
+from typing import List, Optional
```

Comment:
Consider renaming this file to `local_models.py` and extracting the Nautobot-specific models from `nautobot_adapter.py` into a `nautobot_models.py` file? I think this would be a good step in the direction of addressing #52.